src/metadata_tool/tools/parsers.clj

```diff
@@ -21,35 +21,42 @@
             [hickory.select                       :as sel]
             [metadata-tool.sources.confluence     :as cfl]
             [metadata-tool.config                 :as cfg]
+            [clojure.pprint                       :as pp]
             [metadata-tool.sources.metadata       :as md]))
 
 (def not-nil? (complement nil?))
 
 (defn parse-string
   [s]
-  (-> s
-      (str/replace "&nbsp;" " ")
-      (str/replace "\u00A0" " ")
-      str/trim))
+  (if (empty? s)
+    ""
+    (-> s
+        (str/replace "&nbsp;" " ")
+        (str/replace "\u00A0" " ")
+        str/trim)))
+
+(defn resolve-acronym
+  [string]
```

Comment:
for generic string arguments, the convention is `s`.  Great once you get used to it.  
ofc, better as a specific name if you can.  in this case it's an acronym key, maybe just `acronym`?  not sure.